packages/playground/src/views/solutions_view.vue

```diff
@@ -30,33 +30,43 @@ export default {
           "Nostr is a simple, open protocol that enables global, decentralized, and censorship-resistant social media.",
         icon: "nostr.png",
         route: DashboardRoutes.Applications.Nostr,
+        isNew: false,
+        createdAt: 0, // Update creation time in milliseconds ex: 1723468996310 is 'Mon Aug 12 2024 16:23:16'
```

Comment:
If you choose to save the createdAt field inside the card object, I guess you have to hard-code the date in place
```suggestion
        isNew: false,
        createdAt: 1723468996310,
```